### ISSUE TITLE
nyanko build with Wavenet

### ISSUE THREAD

**Issue by G-Wang** [r9y9/deepvoice3_pytorch#100]
Hello, thank you for the great Repo.

I was wondering if it's possible to incorporate the nyanko build to work with wavenet like you've done for deepvoice 3.

Besides modifying the "downsample_step": 4 and "outputs_per_step": 1, what other presets would I have to change to make it work with Wavenet?

**Comment by r9y9** [comment]
> Besides modifying the "downsample_step": 4 and "outputs_per_step": 1, what other presets would I have to change to make it work with Wavenet?

That's it, but I think you will see some problem learning attention with `outputs_per_step` = 1 as I had in #24. I was wondering hybrid attention like Tacotron2 would fix the problem, but haven't had chance to try it yet.

**Comment by G-Wang** [comment]
For Nyanko, the generate_aligned_prediction.py script threw me some errors for size mismatch, I'm guessing since Nyanko predicts down-sampled Mel Spectrogram.

For a rough hack, since Nyanko build still predicts the full linear spectrogram from the converter, I took a trained Nyanko build (500k) predicted linear spectrogram and converted it to a full Mel spectrogram and fed it to your Wavenet-vocoder with the pre-trained model.

![mel](https://user-images.githubusercontent.com/14854468/43240944-dee71ba6-904d-11e8-866d-eced6487588f.png)

Here's the [audio](https://github.com/G-Wang/some_files/raw/master/20180510_mixture_lj_checkpoint_step000320000_ema.wav)

I had to adjust the Nyanko data slightly to match wavenet (removed preemphasis and increased max_position to 2048, and set rescale=True)

The sound quality is not great (very nasally), I'm not sure if it is:
1. removing preemphasis
2. Not fitting Wavenet to new processed data
3. loss of quality from converting from linear to mel
4. Or the model is just not as crisp as your Wavenet + Tacotron 2 (Rayhane-mamah's)

Any thoughts?
